### DISCUSSION TITLE
DAI 2023-2024 - My Docker Compose application - Piemontesi Gwendal

### DISCUSSION THREAD

**Discussion by tasty-orange** [heig-vd-dai-course/heig-vd-dai-course#132] [Show and tell]
[the link](https://github.com/gwendalpiemonte/My-Docker-Compose-Application/tree/main)
![docker](https://github.com/heig-vd-dai-course/heig-vd-dai-course/assets/114938783/91febd9c-e993-4bed-acb3-4c0c9bfb673a)

**Comment by ludelafo** [comment]
Great! Everything looks good to me!
